modules/ROOT/pages/connected-apps-developers.adoc

```diff
@@ -0,0 +1,86 @@
+
+* `full`: Full access to anything the user can do.
+* `read:full`: Read-only access to anything the user can read.
+* `openid`: Read-only access to the user's username and unique Anypoint ID.
+* `profile`: Read-only access to the user's Anypoint profile.
+* `email`: Read-only access to the user's email.
+* `offline_access`: Access to the user's data when they are not logged in.
+
+== OIDC Endpoints and Tokens
+
+To work with OpenID Connect, apps that require access to user data when the user isn’t logged in have to exchange refresh tokens for new access tokens.
+Tokens that are not refreshed expire after 90 days. If the app is using the authorization code grant type, the `offline_access` scope is required to use refresh tokens after original access tokens expire.
+To request a new access token, you must send a request to the token endpoint using the `refresh_token` grant type and the `offline_access` scope.
+When authentication is successful, a refresh token is exchanged for an access token.
+
+The following example shows the structure of a token request:
+
+----
+POST /api/v2/oauth2/token
+Content-Type: application/x-www-form-urlencoded
+grant_type=refresh_token&client_id=<your_client_id>&client_secret=<your_client_secret>
+----
+
+Assuming the token request is fulfilled, the refresh token received in the endpoint response is similar to a regular token response, but it might not contain an `id_token`.
+
+== Sign in with Anypoint Platform Widget
+
+For seamless implementation, developers can integrate a button into the front end of their apps with a pre-built template and set of components. They can add this widget using the following code:
+
+----
+<anypoint-signin scopes="full" redirecturi="https://auth.advancedrestclient.com/oauth-popup.html" clientid="2e38d46b60c5476584cdecba8b516711">
+</anypoint-signin>
+----
+
+For more information about how to embed the button, refer to the https://github.com/anypoint-web-components/anypoint-signin[developer tutorial] and the https://anypoint-web-components.github.io/anypoint-signin/[developer playground].
+
+// == Example
+
+== Create an App
+
+Before you create an app as an organization administrator, consider the following requirements and restrictions:
+
+* Your organization can own up to 200 connected apps.
+* A unique `clientID` is automatically assigned and can’t be modified.
+However, you can modify client secrets in the *Application Settings* page.
+* Your app is owned by your organization; any organization administrator can control it.
+* Actions taken by the app are logged with an application ID and user ID in the audit log so that you can monitor and audit data use.
+
+You must have the organization administrator role to build an application with Connected Apps.
+
+You should also determine the use-case (for example, CI/CD pipeline) and whether the app is internal to your company or is a third-party application that can be distributed outside of your organization.
```

Comment:
I'm not a fan of vague examples. What does CI/CD pipeline mean here? Let's assume I ponder the use-case as instructed here. What do I do with that information?